tool/gravity/cli/clusterupdate.go

```diff
@@ -96,6 +107,65 @@ func newClusterUpdater(
 	return updater, nil
 }
 
+// checkStatus returns an error if the cluster is degraded.
+// If force is true, warnings will be ignored.
+func checkStatus(ctx context.Context, force bool) error {
+	nodes, err := statusapi.NodeStatuses(ctx)
```

Comment:
Like I mentioned above, let's just use `statusapi.FromPlanetAgent(ctx, nil)` here, it returns what you need.